opm/output/data/Wells.hpp

```diff
@@ -414,6 +414,7 @@ namespace Opm {
              well_potential_oil == rate.well_potential_oil &&
              well_potential_gas == rate.well_potential_gas &&
              brine == rate.brine;
+             alq == rate.alq;
```

Comment:
wrong. remove ; on previous line and replace with &&

Reply:
Thanks! Fixed.